frontend/packages/dev-console/src/components/monitoring/overview/MonitoringMetricsSection.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { requirePrometheus } from '@console/internal/components/graphs';
+import MonitoringDashboardGraph from '../dashboard/MonitoringDashboardGraph';
+import { workloadMetricQueries } from './queries';
+import './MonitoringSection.scss';
+
+const WorkloadGraphs = requirePrometheus(({ resource }) => {
+  const ns = resource.metadata.namespace;
+  const workloadName = resource.metadata.name;
+  const workloadType = resource.kind.toLowerCase();
+  return (
+    <>
+      <Grid className="co-m-pane__body">
+        {_.map(workloadMetricQueries, (q, i) => (
+          <GridItem span={12} key={i}>
```

Comment:
I think the Gird layout is not needed here. Because anyway we are displaying a single graph in a row.